student.rb

```diff
@@ -1,12 +1,20 @@
 require './person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    add_classroom(classroom)
```

Comment:
- [ ] there is no need to call the add_classroom method inside the initialize method of Student. please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) as a refresh.